src/includes/configure-scope/python.mdx

```diff
@@ -1,7 +1,12 @@
 ```python
-from sentry_sdk import configure_scope
+import sentry_sdk
 
-with configure_scope() as scope:
+with sentry_sdk.configure_scope() as scope:
     scope.set_tag("my-tag", "my value")
-    scope.user = {'id': 42, 'email': 'john.doe@example.com'}
+    scope.user = {"id": 42, "email": "john.doe@example.com"}
+
+# Or:
+
+sentry_sdk.set_tag("my-tag", "my value")
+sentry_sdk.set_user({"id": 42, "email": "john.doe@example.com"})
```

Comment:
Maybe better examples here would be methods that don't have global helpers:

```py
import logging
import sentry_sdk

with configure_scope() as scope:
    scope.clear()
    scope.set_level(logging.DEBUG)
```

Reply:
I added clear() but set_level already exists. I think we should strive for exposing more stuff as global api rather than documenting the differences.